tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/Stylesheets.spec.ts

```diff
@@ -1,186 +1,280 @@
 ﻿import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
-import {expect} from "@playwright/test";
+import {expect} from '@playwright/test';
 
 test.describe('Stylesheets tests', () => {
-
-  const styleSheetName = 'TestStyleSheetFile';
-  const styleSheetFileName = styleSheetName + ".css";
+  const stylesheetName = 'TestStyleSheetFile.css';
   const ruleName = 'TestRuleName';
-  const styleFolderName = 'TestStylesheetFolder';
+  const stylesheetFolderName = 'TestStylesheetFolder';
 
   test.beforeEach(async ({umbracoUi}) => {
     await umbracoUi.goToBackOffice();
     await umbracoUi.stylesheet.goToSection(ConstantHelper.sections.settings);
   });
 
-  test('can create a stylesheet file', async ({umbracoApi, umbracoUi}) => {
+  test('can create a empty stylesheet', async ({umbracoApi, umbracoUi}) => {
     // Arrange
-    await umbracoApi.stylesheet.ensureNameNotExists(styleSheetFileName);
+    await umbracoApi.stylesheet.ensureNameNotExists(stylesheetName);
 
     //Act
     await umbracoUi.stylesheet.clickActionsMenuAtRoot();
+    await umbracoUi.stylesheet.clickCreateThreeDotsButton();
     await umbracoUi.stylesheet.clickNewStylesheetButton();
-    await umbracoUi.stylesheet.enterStylesheetName(styleSheetName);
     // TODO: Remove this timeout when frontend validation is implemented
-    await umbracoUi.waitForTimeout(1000);
+    await umbracoUi.waitForTimeout(500);
+    await umbracoUi.stylesheet.enterStylesheetName(stylesheetName);
+    // TODO: Remove this timeout when frontend validation is implemented
+    await umbracoUi.waitForTimeout(500);
     await umbracoUi.stylesheet.clickSaveButton();
 
     // Assert
-    expect(await umbracoApi.stylesheet.doesExist(styleSheetFileName)).toBeTruthy();
+    await umbracoUi.stylesheet.isSuccessNotificationVisible();
+    expect(await umbracoApi.stylesheet.doesNameExist(stylesheetName)).toBeTruthy();
     // TODO: when frontend is ready, verify the new stylesheet is displayed under the Stylesheets section
-    // TODO: when frontend is ready, verify the notification displays
 
     // Clean
-    await umbracoApi.stylesheet.ensureNameNotExists(styleSheetFileName);
+    await umbracoApi.stylesheet.ensureNameNotExists(stylesheetName);
```

Comment:
To ensure that this step is executed regardless of whether the test fails, it is recommended to include it within the "afterEach" method. This way, it will be executed after each test.